libpretixsync/src/main/java/eu/pretix/libpretixsync/models/Question.kt

```diff
@@ -0,0 +1,60 @@
+package eu.pretix.libpretixsync.models
+
+import eu.pretix.libpretixsync.check.QuestionType
+import eu.pretix.libpretixsync.db.QuestionLike
+import eu.pretix.libpretixsync.db.QuestionOption
+
+class Question(
+    val id: Long,
+    val serverId: Long,
+    val eventSlug: String?,
+    val position: Long,
+    val required: Boolean,
+    val askDuringCheckIn: Boolean,
+    val showDuringCheckIn: Boolean,
+    val dependencyQuestionServerId: Long?,
+    type: QuestionType,
+    question: String,
+    identifier: String,
+    options: List<QuestionOption>?,
+) : QuestionLike() {
+
+    private val _type: QuestionType = type
+    private val _question: String = question
+    private val _identifier: String = identifier
+    private val _options: List<QuestionOption>? = options
+
+    private var resolveDependencyCalled = false
+    private var resolvedDependency: Question? = null
+
+    override fun getType(): QuestionType = _type
+
+    override fun getQuestion(): String = _question
+
+    override fun getIdentifier(): String = _identifier
+
+    override fun getOptions(): List<QuestionOption>? = _options
+
+    override fun requiresAnswer(): Boolean = required
+
```

Comment:
The implementation of `getDependencyValues` is missing! This breaks dependencies between questions.

Here's a good event for testing:
https://staging.pretix.eu/control/event/postest/questions/questions/

Reply:
Fixed in https://github.com/pretix/libpretixsync/pull/47/commits/bd295fbd2bfa01762a134fffa4660fbd480c9206.

Should we make `getDependency()` and `getDependencyValues()` abstract in `QuestionLike`? Or are there many cases that rely on the defaults?